cares_reinforcement_learning/networks/DQN/network.py

```diff
@@ -4,10 +4,17 @@
 
 
 class Network(nn.Module):
-    def __init__(self, observation_size: int, num_actions: int):
+    def __init__(
+        self,
+        observation_size: int,
+        num_actions: int,
+        hidden_size: list[int] = None,
+    ):
         super().__init__()
+        if hidden_size is None:
+            hidden_size = [1024, 1024]
```

Comment:
The ones we had here previously were just made up I think for the DQNs - it would be good to clarify which should be the default - are you suggesting we make it 512?

Reply:
Given the other ones are 512, make sense to keep them all like that